content/en/docs/concepts/storage/dynamic-provisioning.md

```diff
@@ -17,7 +17,8 @@ calls to their cloud or storage provider to create new storage volumes, and
 then create [`PersistentVolume` objects](/docs/concepts/storage/persistent-volumes/)
 to represent them in Kubernetes. The dynamic provisioning feature eliminates
 the need for cluster administrators to pre-provision storage. Instead, it
-automatically provisions storage when it is requested by users.
+automatically provisions storage when users create
```

Comment:
nit (doesn't block merge): something other than a user might create a PersistentVolumeClaim. I mean technically a ServiceAccount is a user but we don't think of them as such.